src/ArticlePage.jsx

```diff
@@ -0,0 +1,21 @@
+import { useParams } from 'react-router-dom';
+import { fetchArticleById } from './api/api.js';
+import { useEffect, useState } from 'react';
+import Article from './Article.jsx';
+
+export default function articlePage(){
+    const [ article, setArticle ] = useState({});
+    const { article_id } = useParams();
```

Comment:
It would be great if we could have a loading state in here, to give the user feedback when the page is loading 👍